server/src/tree-sitter/analyzer.ts

```diff
@@ -582,6 +580,34 @@ export default class Analyzer {
     return fileUris
   }
 
+  public findFilesInProjectScanner (filePath: string): ElementInfo[] {
+    const parsedPath = path.parse(filePath)
+    let foundFiles: ElementInfo[] = []
+
+    switch (parsedPath.ext) {
+      case '.inc':
+        foundFiles = bitBakeProjectScannerClient.bitbakeScanResult._includes.filter((inc) => {
```

Comment:
in theory, users could use any file extension here, if you follow my other comments, you shouldn't go through the scanner list of files, and instead directly open the URLs provided by bitbake, with the docker URI conversion.

Reply:
>  users could use any file extension here

I am not sure I understand. The case here is `.inc` and I am searching from the include files scanned from `searchFiles()`

> instead directly open the URLs provided by bitbake

As I mentioned in the tracking ticket comment No.20, the `bitbake -e <recipe>` doesn't give useful information for the directive statements like `require busybox.inc`. Or I should say there is not a clear pattern as far as I can see.

Reply:
bitbake -e will always be more accurate than searchFiles. For instance, if multiple buysybox.inc files are present accross layers, you should use the URL provided by bitbake -e to know which one is used. Hence why I think you should refactor this part. When presenting URIs after bitbake -e, don't use the results of searchFiles stored in the project scanner, user the URLs from bitbake (but call resolveContainerPath())

Reply:
I see the point. Just like i said above, there is no clear pattern that shows me how to apply the `bitbake -e` outputs . 
An example: 
![image](https://github.com/yoctoproject/vscode-bitbake/assets/47988425/7c960e1a-ba70-49f0-b288-b9a7fc613f30)
The `acpid.inc` is not mentioned in the `INCLUDE HISTORY` at the top of the command output. Though it is mentioned in the operations above some variables. I don't think I should check every variable's operations just to see that `acpid.inc` is actually used.